addons/hr_appraisal/static/src/css/hr_appraisal.css

```diff
@@ -0,0 +1,7 @@
+.oe_appraisal_kanban_box {
+    height: auto;
+    width: 300px;
+}
+.openerp .oe_stat_button:hover .oe_form_x2many_counter a{
```

Comment:
don't fix x2many_counter widget color issue here. fix it in base.css for x2man_counter or may be we can create new backlog.